src/ptvsd/server/attach.py

```diff
@@ -83,26 +69,23 @@ def enable_attach(address=(DEFAULT_HOST, DEFAULT_PORT), redirect_output=None, lo
     ptvsd.server.log.to_file()
     ptvsd.server.log.info('enable_attach{0!r}', (address,))
 
-    if redirect_output is not None:
-        ptvsd.server.log.info('redirect_output deprecation warning.')
-        warnings.warn(_redirect_output_deprecation_msg, DeprecationWarning, stacklevel=2)
-
     if is_attached():
         ptvsd.server.log.info('enable_attach() ignored - already attached.')
-        return
-
-    dbg = get_global_debugger()
-    if bool(dbg):
-        dbg.dap_debugger_attached.clear()
+        return None, None
 
     # Ensure port is int
-    port = address[1]
-    address = (address[0], port if type(port) is int else int(port))
+    host, port = address
+    address = (host, port if type(port) is int else int(port))
```

Comment:
No need to check for type - `int()` is a no-op if it's already an int. Not sure why it was doing it like that before...